mass_editing/models/mass_object.py

```diff
@@ -5,6 +5,16 @@
 from odoo import api, fields, models, _
 
 
+class Fields(models.Model):
+    _inherit = 'ir.model.fields'
+
+    @api.model
+    def search(self, args, offset=0, limit=0, order=None, count=False):
+        print args
```

Comment:
I guess this is here for debugging and you will remove it. Also, this is causing travis to fail.